### PR TITLE
add jupyter notebook example

### PR DESCRIPTION
**Pull request by mxyng**

### CODE REVIEW

File: examples/jupyter-notebook/ollama.ipynb [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+  {
+   "cell_type": "markdown",
+   "id": "38d57674-b3d5-40f3-ab83-9109df3a7821",
+   "metadata": {},
+   "source": [
+    "# Ollama Jupyter Notebook\n",
+    "\n",
+    "Ollama is the easiest way to run large language models (LLMs) locally. You can deploy it to macOS by installing the the macOS application, Linux by running the install script (below), and Docker or Kubernetes by pulling the official Ollama Docker image.\n",
+    "\n",
+    "For best results, this notebook should be run on a Linux node with GPU or an environment like Google Colab."
```
Comment (BruceMacD) [inline examples/jupyter-notebook/ollama.ipynb:12]:
```suggestion
    "For best results, this notebook should be run on a Linux node with a GPU or an environment like Google Colab."
```

**Review by BruceMacD — COMMENTED** [review]
Looking good, I feel like this is a missing some explanation around how to use Ollama with Colab. It looks like my goal is to run Ollama in a notebook so that I can call it externally for inference on a GPU?

**Comment by mxyng** [comment]
> It looks like my goal is to run Ollama in a notebook so that I can call it externally for inference on a GPU?

That's been the most common use case of Ollama in Colab I've seen so far: run Ollama with GPU then expose it with ngrok so I can access it remotely

**Review by BruceMacD — APPROVED** [review]
Might be nice to add a README that explains why you would want to run Ollama in a notebook. Otherwise, this looks good.

**Comment by raffaelemancuso** [comment]
Where is this `examples/jupyter-notebook/ollama.ipynb` nowadays? It's not in ollama-python

**Pull request merged**